### DISCUSSION TITLE
Package install failed

### DISCUSSION THREAD

**Discussion by jangx147** [jamesjiadazhan/dietaryindex#282] [Q&A] [answered]
Hello Jiada, 

Thank you for sharing the dietaryindex R package. 
I hope to utilize this package for my research, but I am having trouble loading the package.
I tried both directions under installation, but I am getting the following error message:

_Error: Failed to install 'unknown package' from GitHub:
HTTP error 401. Bad credentials Rate limit remaining: 53/60 Rate limit reset at: 2024-05-20 03:06:30 UTC_

Can you please help me troubleshoot this? 

Thank you,
Subin

**Comment by jamesjiadazhan** [comment, marked as answer]
You can download the package manually:
![image](https://github.com/jamesjiadazhan/dietaryindex/assets/108076575/a1199f74-42c9-45a2-b177-b63d31040e86)

Un zip the file

And then use the following codes to install it locally:
# load the package downloaded from github 
install.packages("/Users/yan/Downloads/dietaryindex-main", repos = NULL, type = "source")

I hope it is helpful. Let me know if you have other questions.

If possible, please give us a Star on GitHub.
<img width="1326" alt="image" src="https://github.com/jamesjiadazhan/dietaryindex/assets/108076575/ac98fdbd-d043-4172-9ca5-b396e7bde564">

Also, if you publish your work, please cite us:

- Jiada James Zhan, Rebecca A Hodge, Anne Dunlop, et al. Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies. bioRxiv. Published online August 07, 2023:2023.08.07.548466. doi:10.1101/2023.08.07.548466
  - https://www.biorxiv.org/content/10.1101/2023.08.07.548466v2
